content/applications/sales/point_of_sale/shop/ship_later.rst

```diff
@@ -0,0 +1,44 @@
+==========
+Ship later
+==========
+
+The **Ship Later** feature allows you to sell products and schedule delivery at a later date. This
+feature comes in handy when a product is out of stock, so voluminous that it requires to be
+shipped, or if, for any reason, the customer needs their order to be shipped later, etc.
+
+Configuration
+=============
+
+:ref:`Go to the POS settings <configuration/settings>`, scroll down to the :guilabel:`Inventory`
+section, and enable :guilabel:`Ship Later`.
+
+.. image:: ship_later/settings.png
+   :scale: 85%
+
+Once activated, you can:
+
+- Choose the location from where the products are shipped by completing the :guilabel:`Warehouse`
+  field.
+- Define a :guilabel:`Specific route`, or leave this field empty to use the default route.
+- Define the :guilabel:`Shipping Policy`; select :guilabel:`As soon as possible` if the products
+  can be delivered separately or :guilabel:`When all products are ready` to ship all the products at
+  once.
```

Comment:
You didn't think I'd leave a review without one of my famous dumb questions, did you? :smile: so here it is: what defines "ready" exactly?

Reply:
When all the products are available -
For example: the time to be delivered differs, you only send the products in one delivery when they have all arrived.